checkov/common/util/prompt.py

```diff
@@ -0,0 +1,273 @@
+                                                            "default": "azurerm_policy_definition"
+                                                        },
+                                                    }
+                                                ]
+                                            }
+                                        },
+                                        {  # GCP
+                                            "context": {
+                                                "prompt_if": 'gcp',
+                                                "text": 'Select a terraform object for this check',
+                                                "typ": click.Choice(['resource']),
+                                                "default": "resource",
+                                                "sub_prompts": [
+                                                    {
+                                                        "supported_resource": {
+                                                            "prompt_if": 'resource',
+                                                            "text": 'Enter the terraform object type',
+                                                            "typ": str,
+                                                            "default": "google_project_iam_policy"
+                                                        },
+                                                    }
+                                                ]
+                                            }
+                                        },
+                                    ]
+                                }
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+    }
+
+    def __init__(self):
+        print(checkov_banner)
+        self.responses = {}
+        self.prompt()
+
+    def go(self):
+        print("Please ensure you are at the root of the Checkov repository before completing this prompt")
+
+        # Load our prompts
+        module = importlib.import_module("checkov.common.util.prompt")
+
+        # Load an instance of our Check class (this could be expanded to do more later)
+        instance = getattr(module, "Check")(self.responses)
+
+        # Call the action on that Class (add, remove, modify - leaves room for more actions)
+        getattr(instance, self.action)()
+
+    # Recurse over our prompt, populating new class attributes from the keys and
+    # user-supplied answers
+    def prompt(self, prompt_map=PROMPTS, prompt_if=None):
+        for k, v in prompt_map.items():
+            if "prompt_if" not in v or v["prompt_if"] == prompt_if:
+                # Prompt the user
+                p = click.prompt(v["text"], type=v["typ"], default=v["default"])
+
+                # Create the action on our object
+                if k is 'action':
```

Comment:
```suggestion
                if k == 'action':
```
you want to check the string equality not the identity